onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -613,6 +613,10 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
               "The value to be filled in the attention mask. Default value is -10000.0f",
               AttributeProto::FLOAT,
               OPTIONAL_VALUE)
+        .Attr("output_qk",
```

Comment:
Do not need this attribute. You can deduce the flag from whether there is 3rd output